components/cloud/azure/src/token_credentials_with_cache/utils.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::sync::{Arc, Mutex, RwLock};
+
+use azure_core::auth::TokenResponse;
+use time::OffsetDateTime;
+
+/// Refresh time to use in seconds
+pub(crate) const DEFAULT_REFRESH_TIME: i64 = 300;
+/// Timeout on refreshing token in seconds
+const DEFAULT_REFRESH_TOKEN_TIMEOUT: i64 = 10;
+
+fn config_default_refresh_token_timeout() -> i64 {
+    fail::fail_point!("mock_token_cache::refresh_timeout", |_| 1);
```

Comment:
```suggestion
    fail::fail_point!("on_config_default_refresh_token_timeout", |_| 1);
```

In tikv, we prefer name failpoints in the style `[on_]func_name[_verb]`, "[ ]" means optional.

Reply:
Skipped.